addons/compat_rf/compat_rf_wardrobe/config.cpp

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class SUBADDON {
+        name = COMPONENT_NAME;
+        author = ECSTRING(common,ACETeam);
+        authors[] = {"OverlordZorn [CVO]"};
+        url = ECSTRING(main,URL);
+        
+        VERSION_CONFIG;
+        addonRootClass = QUOTE(ADDON);
+        requiredAddons[] = {
+            "ace_wardrobe",
+            "RF_Data_Loadorder"
+        };
+        skipWhenMissingDependencies = 1;
+
+        requiredVersion = REQUIRED_VERSION;
+       
+        units[] = {};
+        weapons[] = {};
+    };
+};
```

Comment:
General convention for these
```suggestion
class CfgPatches {
    class SUBADDON {
        name = COMPONENT_NAME;
        units[] = {};
        weapons[] = {};
        requiredVersion = REQUIRED_VERSION;
        requiredAddons[] = {
            "ace_wardrobe",
            "RF_Data_Loadorder"
        };
        skipWhenMissingDependencies = 1;
        author = ECSTRING(common,ACETeam);
        authors[] = {"OverlordZorn [CVO]"};
        url = ECSTRING(main,URL);
        VERSION_CONFIG;
        addonRootClass = QUOTE(ADDON);
    };
};
```